styleguide

```diff
@@ -244,12 +244,63 @@ module Styleguide
     end
   end
 
+  class Simplecov < Thor::Group
+    include Helpers
+
+    class_option :defaults, default: true, type: :boolean, desc: 'Whether to import the default configuration'
```

Comment:
I'm not sure what this option means, I just left it to the default (`true`) but why wouldn't I want to import the defaults?